docs/distributing-an-application/airgapped-installations.md

```diff
@@ -59,3 +59,10 @@ page.
 ## Adding Additional Nodes
 
 In order to add additional nodes to your cluster, just navigate to the Cluster page of the Admin Console, click the "Add Node" button, and follow the instructions there. For more detailed instructions see the [add nodes](/distributing-an-application/add-nodes/) page of the docs.
+
+## Imageless Airgap Installations
+Licenses with Airgap installations enabled can additionally be configured to offer imageless bundles to customers.
+When imageless downloads are enabled in the vendor portal, customers will see two links at the download URL you
```

Comment:
I think we don't do second person pronouns.  It's in the contributing guide somewhere (linked from the README file)